scripts/kickstart_generator.rb

```diff
@@ -23,17 +23,17 @@ def initialize(build_base, targets, puddle, manageiq_checkout, appliance_checkou
       @ssui_checkout      = ssui_checkout
     end
 
-    def run(task = :all)
+    def run
```

Comment:
This change seems to be breaking a bunch of tests that expect run to receive `task` as an arg.